media/css/mozorg/mpl-1-1.scss

```diff
@@ -2,7 +2,10 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../pebbles/includes/lib';
+$font-path: '/media/fonts';
```

Comment:
This file and the other mpl .scss files don't need to include the library. Not a blocker but a bit tidier.